### ISSUE TITLE
Nushell error `can't convert string to range`

### ISSUE THREAD

**Issue by zephraph** [ellie/atuin#848]
<table>
<tr>
	<td>Program
	<td>Version
<tr>
	<td>Nushell
	<td>0.78.0
<tr>
	<td>starship
	<td>1.13.1 (w/ fixes for Nu)
<tr>
	<td>atuin
	<td>14.0.0
</table>

```
Error: nu::shell::cant_convert

  × Can't convert to range.
   ╭─[entry #7:1:1]
 1 │ # 8b047401-1bbe-4183-bff1-f85388ebb5dd
 2 │ commandline (RUST_LOG=error run-external --redirect-stderr atuin search "--shell-up-key-binding" "--interactive" "--" (commandline) | complete | $in.stderr | str substring ',-1')
   ·                                                                                                                                                                             ──┬──
   ·                                                                                                                                                                               ╰── can't convert string to range
   ╰────
```

This is happening in `init.nu` on line 33. If I drop the `str substring ',-1' it stops failing but it does tend to collect newlines at the end of the completion as show in the gif below. 

![atuin-bug](https://user-images.githubusercontent.com/3087225/229960475-cc4d90d2-59f7-469f-80a0-247759d84bc7.gif)

**Comment by zephraph** [comment]
I believe it's due to [this](https://github.com/nushell/nushell/pull/8660) (which is another breaking change snuck into this release 😅) . It looks like `str substring ',-1'` has to be changed to `str substring ..-1` like [this example](https://github.com/nushell/nushell/pull/8660/files#diff-0d7b2f49835cc241b18b44f73b283a244e0ae51ad0db982e4b2acc5a652674e6L334-R334)

**Comment by zephraph** [comment]
Ah, yeah, this is fixed by #840 which I totally missed.

**Comment by ellie** [comment]
Thank you for reporting this 🙏
